packages/aws-cdk-lib/aws-events-targets/test/ecs/event-rule-target.test.ts

```diff
@@ -1095,3 +1096,100 @@ test.each([
     ],
   });
 });
+
+test('Non-imported TaskDefinition role is targeting by Ref', () => {
```

Comment:
nit: could you change the verbiage here? Based on the name, I'm not certain what it's testing.